### ISSUE TITLE
Question about Scm Bot buying speed

### ISSUE THREAD

**Issue by Alex1os** [Maxx53/SCMBot#414]
Hello again,

I was wondering about the buying and scanning speed of the bot.
What I would love to know from Maxx or someone else is, do ping times acutally influence the bot's buying speed and scanning? 
As what I've learned in the past months, the scanning speed is changeable in the bot's settings but is there are way to make the bot react faster when it comes to buying an item?

thanks in advance 

Lg Alex

**Comment by mtzi** [comment]
The russians on the post below yours say that it doesnt buy as well, and noticed the $ sign next to prices. I personally use Euros, and its the first time i noticed a $ next to scan prices in SCMbot. Hopefully its a bug, and maxx will look into it

**Comment by vampig** [comment]
yep, buddy, you're doing right. bot is stopped buying from yesterday, maybe problem in cookies.. don't know for sure.

**Comment by Alex1os** [comment]
oh my god thx guys, that's explaines everything!

my bot also started showing Dollar instead of Euros and he doesn't buy any items even if I change the wished price to how much dollar is worth...

i get the error messages: "There was a problem buying your item. The listing may have been removed.  Refresh the page and try again." and "unknown error"

@Maxx we really need ur help mate :-)

Lg Alex

**Comment by Maxx53** [comment]
https://github.com/Maxx53/SCMBot/issues/415

**Comment by Randjolaf** [comment]
Regarding that, I have my SCM bot up 24 hours a day and yet I go later on the steam market, check my item, and I missed a literally free item! 0.03 euros insteand of 10. 
How does this happen? I am so confused. Sometimes I wonder who would even put something like that on the market.

**Comment by vampig** [comment]
as usual, very cheap items comes from hacked accounts. be careful. your success may become your fail.
we're all really be happy to get this cheap things, but: 1st, this item is one of the millions; 2nd, your bot is one(two, three, does'nt matter) of the 100ks other bots... life is life, na-na na-na-na.

**Comment by Randjolaf** [comment]
The item is Dota 2 International compendium. It can be officially bought in-game for 9.99$/9.29€. Though I checked google for currency it said 9.10€, though maybe it's also more expencive in US too, because of some taxes or something. 

What do you mean by "your success may become your oblivion"? What could possibly happen? I would just buy that account and the person who sold it will know my account name. What can he do with it?
